src/slowmokit/methods/metrics/classification_report.hpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file methods/metrics/classification_report.hpp
+ *
+ * Easy include to show the classification report of data
+ */
+
+#ifndef SLOWMOKIT_CLASSIFICATION_REPORT_HPP
+#define SLOWMOKIT_CLASSIFICATION_REPORT_HPP
+#include "../../core.hpp"
+
+/**
+ * Takes predicted and actual values of classes
+ * @param predictedValue -> predicted values 
+ * @param trueValue -> true values
+ * @returns the classification report
+ * @throws domain_error exception when size of the two vectors is not equal
+ */
+
+template <class T>
+class ClassificationReport
+{
+private:
+    std::vector<T> trueValue;
+    std::vector<T> predictedValue;
+    std::map<T, T> classes, truePositive, falsePositive, trueNegative, falseNegative;
```

Comment:
Change these maps to map<T, int> as count will always be an integer no matter what input arrives